### ISSUE TITLE
The number of Movi-e dataset

### ISSUE THREAD

**Issue by XiongyiCai** [google-research/kubric#286]
Hi, when I use the Kubric(Movi-E), I find the number of the train set is about 9750, the validation set is about 250 and the test set is about 999, which is different with the number 38,325/799 for train/validation in paper "TAP-Vid: A Benchmark for Tracking Any Point in a Video ".

**Comment by ZHAOZHIHAO** [comment]
Probably the number 38,325 contains movi-c,d,e,f because all of them are generated from Kubric.  And the numbers for TAP-Vid-RGB-Stacking contains movi-a,b.

**Comment by YunhengWu-IB** [comment]
Hi, @XiongyiCai I’m also confused by the same question. Have you found the answer to it now?
